### DISCUSSION TITLE
Best way to not have html in the exported markdown?

### DISCUSSION THREAD

**Discussion by nickanderson** [kaushalmodi/ox-hugo#692] [Q&A] [answered]
Our hugo build process strips html from the markdown files in order to prevent potentially malicious stuff. Right now I just manually delete html that lands in my exports, but I am sure someone has a little snippet that I could save a bunch of time with.

**Comment by kaushalmodi** [comment, marked as answer]
`ox-hugo` relies on having HTML in the exported Markdown to match a lot of features that Org Mode has.

It's not possible to export HTML-free Markdown.

> Our hugo build process strips html from the markdown files in order to prevent potentially malicious stuff.

To truly protect your site from malicious attacks, look into CSP (content security protection) headers. 

I have this setting in the hugo config for all my sites for many years (and I have CSP enabled): https://ox-hugo.scripter.co/doc/goldmark/#enable-unsafe-html . That will prevent Hugo from stripping away the HTML.

**Reply by nickanderson** [reply]
> It's not possible to export HTML-free Markdown.

This is what I was trying to achieve.

I don't care that `hugo` strips html, I care that I don't have to manually clean up the markdown I export from org so that other people are not distracted by html inside files I exported. Most others working on that site are writing directly in markdown from the start.

**Comment by kaushalmodi** [comment]
> Right now I just manually delete html that lands in my exports, but I am sure someone has a little snippet that I could save a bunch of time with.

If you just want to remove all the HTML, Hugo already does that by default.. which is this:

```conf
[markup.goldmark.renderer]
  unsafe = false
```

.. but proceed with that at your own risk; you will find a lot of features added by `ox-hugo` to not work if you do that.
